### PR TITLE
Adds the front end files required to interact with the Users Authenti…

### PR DESCRIPTION
**Pull request by zyelcrier**
…cation process  APIs for review.
Caleb's Branch I opened it due to technical difficulties  he cant view github on his school wi-fi currently
image
image
image

### CODE REVIEW

File: capstone/src/main/webapp/badlogin.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <!--Let browser know website is optimized for mobile-->
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+  </head>
+  
+  <body onload="invalidEmailRequest()">
+    <div class= "root-layout">
+        <nav class="nav-wrapper">
+            <a href="/" class="brand-logo">Studentia</a>
+            <a href="#" class="sidenav-trigger" data-target="slide_out" ><i class="material-icons">menu</i></a>
```
Comment (ljfranklin) [inline capstone/src/main/webapp/badlogin.html:21]: `slide_out` to `slide-out`, same on other page

File: capstone/src/main/webapp/dashboard.html [hunk 2]
```diff
@@ -13,9 +13,70 @@
...
+        <a href="" class="brand-logo">Studentia</a>
+        <a href="#" class="sidenav-trigger" data-target="slide_out" ><i class="material-icons">menu</i></a>
+        <ul id="nav-mobile" class="right hide-on-med-and-down">
+          <li><a href="findStudySet.html">Search</a></li>
+          <li><a href="/">Home</a></li>
+        </ul>
+    </nav>
+
+    <ul class="sidenav" id="slide_out">
+          <li><a href="findStudySet.html">Search</a></li>
```
Comment (ljfranklin) [inline capstone/src/main/webapp/dashboard.html:28]: Run formatter to fix indents

File: capstone/src/main/webapp/dashboard.html [hunk 3]
```diff
@@ -13,9 +13,70 @@
...
+           © 2020 Studentia inc
+       </div>
+    </footer>
+   </div>
      <!--JavaScript at end of body for optimized loading-->
-     <script type="text/javascript" src="js/materialize.min.js"></script>
+    <script type = "text/javascript" src = "https://code.jquery.com/jquery-2.1.1.min.js"></script>         
+    <script type="text/javascript" src = "https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script> 
+    <script src="script.js"></script>  
+    <script src="script-for-user-authentication-process.js"></script>
```
Comment (ljfranklin) [inline capstone/src/main/webapp/dashboard.html:80]: I think you need to add this JS file to the PR

Comment (Cayle) [reply capstone/src/main/webapp/dashboard.html:80]: Okay, I didn't add it at first because it's about 200 lines. Is it okay if I add it now ?

**Pull request merged**